File: unittest/scripts/auto/js_shell/scripts/util_help_norecord.js
//@ util help
util.help();

//@ util help, \? [USE:util help]
\? util

//@ util checkForServerUpgrade help
util.help('checkForServerUpgrade');

//@ util checkForServerUpgrade help, \? [USE:util checkForServerUpgrade help]
\? checkForServerUpgrade

// WL13807-TSFR_1_1
//@ util dumpInstance help
util.help('dumpInstance');

//@ util dumpInstance help, \? [USE:util dumpInstance help]
\? dumpInstance

// WL13807-TSFR_2_1
//@ util dumpSchemas help
util.help('dumpSchemas');

//@ util dumpSchemas help, \? [USE:util dumpSchemas help]
\? dumpSchemas

//@ util importJson help
util.help('importJson');

//@ util importJson help, \? [USE:util importJson help]
\? importJson

//@ util configureOci help {__with_oci == 1}
util.help('configureOci');

//@ util configureOci help, \? [USE:util configureOci help] {__with_oci == 1}
\? configureOci

//@ oci help {__with_oci == 1}
\? oci

//@ util importTable help
util.help('importTable');

//@ util importTable help, \? [USE:util importTable help]
\? importTable

File: unittest/scripts/auto/py_shell/scripts/util_help_norecord.py
#@ util help
util.help()

#@ util help, \? [USE:util help]
\? util

#@ util check_for_server_upgrade help
util.help('check_for_server_upgrade')

#@ util check_for_server_upgrade help, \? [USE:util check_for_server_upgrade help]
\? check_for_server_upgrade

#@ util copy_instance help
# WL15298_TSFR_1_1
# WL15298_TSFR_4_4_1
# WL15298_TSFR_5_1_1
util.help('copy_instance')

#@ util copy_instance help, \? [USE:util copy_instance help]
# WL15298_TSFR_1_1
# WL15298_TSFR_4_4_1
# WL15298_TSFR_5_1_1
\? copy_instance

#@ util copy_schemas help
# WL15298_TSFR_2_1
# WL15298_TSFR_4_4_1
# WL15298_TSFR_5_1_1
util.help('copy_schemas')

#@ util copy_schemas help, \? [USE:util copy_schemas help]
# WL15298_TSFR_2_1
# WL15298_TSFR_4_4_1
# WL15298_TSFR_5_1_1
\? copy_schemas

#@ util copy_tables help
# WL15298_TSFR_4_4_1
util.help('copy_tables')

#@ util copy_tables help, \? [USE:util copy_tables help]
# WL15298_TSFR_4_4_1
\? copy_tables

#@ util dump_binlogs help
util.help('dump_binlogs')

#@ util dump_binlogs help, \? [USE:util dump_binlogs help]
\? dump_binlogs

# WL13807-TSFR_1_1
#@ util dump_instance help
util.help('dump_instance')

#@ util dump_instance help, \? [USE:util dump_instance help]
\? dump_instance

# WL13807-TSFR_2_1
#@ util dump_schemas help
util.help('dump_schemas')

#@ util dump_schemas help, \? [USE:util dump_schemas help]
\? dump_schemas

# WL13804-TSFR_6_1
#@ util dump_tables help
util.help('dump_tables')

#@ util dump_tables help, \? [USE:util dump_tables help]
\? dump_tables

# WL13804-TSFR_1_4
#@ util export_table help
util.help('export_table')

#@ util export_table help, \? [USE:util export_table help]
\? export_table

#@ util import_json help
util.help('import_json')

#@ util import_json help, \? [USE:util import_json help]
\? import_json

#@ util import_table help
util.help('import_table')

#@ util import_table help, \? [USE:util import_table help]
\? import_table

#@ util load_binlogs help
util.help('load_binlogs')

#@ util load_binlogs help, \? [USE:util load_binlogs help]
\? load_binlogs

#@ util load_dump help
util.help('load_dump')

#@ util load_dump help, \? [USE:util load_dump help]
\? load_dump

#@ util debug collect_diagnostics (full path)
\? util.debug.collect_diagnostics

#@ util debug collect_diagnostics with util.debug.help (partial path)
util.debug.help("collect_diagnostics")

#@ util debug collect_high_load_diagnostics
\? collect_high_load_diagnostics

#@ util debug collect_slow_query_diagnostics
\? collect_slow_query_diagnostics

#@ Help on remote storage
\? remote storage

#@ Help on mysql_native_password
\? mysql_native_password